.spi.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+builder:
+   configs:
+   - documentation_targets: ["CareKit", "CareKitUI", "CareKitStore", "CareKitFHIR"]
+     platform: ios
+   - platform: ios
+     scheme: "CareKit"
+   - platform: watchos
+     scheme: "CareKit"
```

Comment:
It looks like SPI automatically infers a configuration based on our repo, can you walk me through why this file is explicitly needed?

Reply:
It can most of the time, but sometimes it can’t. The latest changes you pushed combined the targets, but it’s possible SPI can still make a mistake because they’re multiple frameworks in one. You can see the builds for CareKit here: https://swiftpackageindex.com/carekit-apple/CareKit/builds

Also, the docs won’t build without being told specifically to do so in the spi file

Reply:
Let me know if need me to answer more questions from the comment added here: https://github.com/carekit-apple/CareKit/pull/691/files#r1162276983